website/styles/_typo.scss

```diff
@@ -0,0 +1,54 @@
+@import 'variables';
+
+:root {
+  --fs-display: 3.125rem;
+  --fs-700: 2.375rem;
+  --fs-600: 1.8125rem;
+  --fs-400: 1.375rem;
+  --fs-300: 1rem;
+  --fs-100: 0.75rem;
+}
+
+@media (min-width: $breakpoint-medium) {
+  :root {
+    --fs-display: 3.75rem;
+    --fs-700: 2.875rem;
+    --fs-700: 2.875rem;
+    --fs-600: 2.375rem;
+    --fs-400: 1.8125rem;
+    --fs-300: 1.125rem;
+    --fs-100: 0.875rem;
+    --fs-050: 0.75rem;
+  }
+}
+
+body {
+  line-height: 1.5;
+}
+
+.io-layout {
+  h1,
+  h2,
+  h3,
+  h4,
+  h5 {
+    // Help prevent overflow of long words/names/URLs
+    word-break: break-word;
+
+    // Optional, not supported for all languages:
+    hyphens: auto;
+
+    // Clear top margin
+    margin-top: 0;
+    margin-bottom: 0.65em;
+    line-height: calc(2px + 2ex + 2px);
```

Comment:
`4px + 2ex`

Reply:
Je me suis basé sur cette article pour définir le lien-height.
https://moderncss.dev/generating-font-size-css-rules-and-creating-a-fluid-type-scale/#line-height-and-vertical-spacing